### PR TITLE
app-text/dblatex: Add Py3 compatible 0.3.11, drop old stable 0.3.7

### PR DESCRIPTION
**Pull request by zpuskas**
I've kept app-text/dblatex 0.3.10-r1 for now, as it's in stable tree and don't want to break users. If there are no breakages with 0.3.11, then in about a month I'll submit a STABLEREQ for 0.3.11 and will drop 0.3.10 to finish migrating away from Python 2.

### CODE REVIEW

**Comment by zpuskas** [comment]
I'll fix the QA issue later today.

File: app-text/dblatex/dblatex-0.3.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+PATCHES=(
+	 "${FILESDIR}/${P}-path-logging.patch"
+	 "${FILESDIR}/${P}-setup.patch"
+)
+
+src_unpack() {
+	unpack ${A}
+	# Remove python3 "versioning" in the directory name
+	mv "${S}py3" "${S}"
+}
```
Comment (juippis) [inline app-text/dblatex/dblatex-0.3.11.ebuild]: You can just define `S=` globally to avoid this.

File: app-text/dblatex/dblatex-0.3.11.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_{6,7} )
```
Comment (juippis) [inline app-text/dblatex/dblatex-0.3.11.ebuild]: One empty line between these please.

File: app-text/dblatex/dblatex-0.3.11.ebuild [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+inherit distutils-r1
+
+DESCRIPTION="Transform DocBook using TeX macros"
+HOMEPAGE="http://dblatex.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}py3.tar.bz2"
```
Comment (juippis) [inline app-text/dblatex/dblatex-0.3.11.ebuild]: You can just use the downloads... address, because this mirror doesn't do much better than that.

**Comment by zpuskas** [comment]
All comments fixed as well as the QA issue found by CI.

**Pull request closed without merging**